x-pack/plugins/monitoring/public/alerts/callout.tsx

```diff
@@ -5,78 +5,105 @@
  */
 
 import React, { Fragment } from 'react';
-import { i18n } from '@kbn/i18n';
-import { EuiCallOut, EuiSpacer } from '@elastic/eui';
-import { CommonAlertStatus } from '../../common/types/alerts';
-import { AlertSeverity } from '../../common/enums';
+import {
+  EuiPanel,
+  EuiSpacer,
+  EuiAccordion,
+  EuiListGroup,
+  EuiListGroupItem,
+  EuiTextColor,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+} from '@elastic/eui';
 import { replaceTokens } from './lib/replace_tokens';
 import { AlertMessage, AlertState } from '../../common/types/alerts';
-
-const TYPES = [
-  {
-    severity: AlertSeverity.Warning,
-    color: 'warning',
-    label: i18n.translate('xpack.monitoring.alerts.callout.warningLabel', {
-      defaultMessage: 'Warning alert(s)',
-    }),
-  },
-  {
-    severity: AlertSeverity.Danger,
-    color: 'danger',
-    label: i18n.translate('xpack.monitoring.alerts.callout.dangerLabel', {
-      defaultMessage: 'Danger alert(s)',
-    }),
-  },
-];
+import { AlertsByName } from './types';
+import { isInSetupMode } from '../lib/setup_mode';
+import { SetupModeContext } from '../components/setup_mode/setup_mode_context';
+import { AlertConfiguration } from './configuration';
 
 interface Props {
-  alerts: { [alertTypeId: string]: CommonAlertStatus };
+  alerts: AlertsByName;
   stateFilter: (state: AlertState) => boolean;
```

Comment:
We should completely remove `stateFilter` (and its logic) from the `<AlertsCallout />` component, since we only use it on node detail pages (which are already filtered by the server)

Reply:
done